bin/install_dependencies.sh

```diff
@@ -1,18 +1,11 @@
 #!/bin/bash
 
-if ! [ -x "$(command -v pandoc)" ]; then
-  echo "Pandoc not found. Installing..."
-  wget https://github.com/jgm/pandoc/releases/download/2.2.2.1/pandoc-2.2.2.1-1-amd64.deb -O /tmp/pandoc.deb
-  sudo dpkg -i /tmp/pandoc.deb
-else
-  echo "Pandoc already installed."
-fi
+if [ -x "$(command -v apt)" ]; then
+  echo "Installing the necessary dependencies"
 
-if ! [ -x "$(command -v wkhtmltopdf)" ]; then
-  echo "wkhtmltopdf not found. Installing..."
-  # Copied from https://github.com/thewca/worldcubeassociation.org/blob/5698831f9addb26091641c80beadfa7c902bcb18/chef/site-cookbooks/wca/recipes/regulations.rb#L17-L18
-  wget https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.4/wkhtmltox-0.12.4_linux-generic-amd64.tar.xz -O /tmp/wkhtml.tar.xz
-  sudo tar -xf /tmp/wkhtml.tar.xz --strip-components=1 -C /usr/local
+  apt update &&
+  # fonts-liberation includes Liberation Sans, which is used in this repo, and fonts-noto is the fallback for wider language support
+  apt install -y wget fonts-liberation fonts-noto pandoc weasyprint
```

Comment:
The reason why we did not use `apt` before is that it allows us to pin the specific version of `pandoc` and `wkhtmltopdf`. That way, unexpected breaking changes can be avoided.
Your new `apt` way looks more convenient on the surface, but actually it will always just take the newest version that is available at the time of running the script. In six months or one year, that might be a new major version that suddenly needs new CLI args to be passed, or otherwise the output looks horrible. (Like, say for example the CLI parameter for defining "use this CSS file please" changes from `--css $some_css_file` to `--stylesheet $some_css_file`. In the "best" case scenario, the old `weasyprint` invocation will fail because then `--css` becomes an unrecognized CLI option. In the "worst" case, it will just continue, silently skipping the `--css` option, which means our documents get compiled without stylesheet and "suddenly" look awful and everybody will scratch their heads what happened because these issues are hard to debug.

Reply:
TL;DR: Is there a way for you to pin specific versions of at least `pandoc` and `weasyprint`?
(The font files don't need pinned versions because we don't invoke CLI commands on them that could break in the future)

Reply:
Good point there. I think the best option would be to specify the package versions using apt. I'll make this change.